### ISSUE TITLE
windows 10 error

### ISSUE THREAD

**Issue by tonylu110** [GregVido/mica-electron#40]
![b52927cda4ef3b92e82db44f29ced5db](https://github.com/GregVido/mica-electron/assets/58923223/eeb75cfd-10e3-4880-a80c-daa79d15ea86)
like this #38 

[It's my repo](https://github.com/tonylu110/uyou-todo-electron)

**Comment by GregVido** [comment]
I updated, did that solve the problem?

**Comment by tonylu110** [comment]
> I updated, did that solve the problem?

do you push to npm?

**Comment by GregVido** [comment]
yes

**Comment by GregVido** [comment]
(it's long I have internet problems)

**Comment by tonylu110** [comment]
> (it's long I have internet problems)

![c765d3114472ee304d78005cd89ba6a7](https://github.com/GregVido/mica-electron/assets/58923223/75b5c6e1-795e-4257-afa7-8f1b26956cd7)

also can't run
I use your demo, it's can't run

also error
https://github.com/GregVido/mica-electron/blob/24408291a0f5567fb66bc1b142d68e5db6580802/module/app.cpp#L126

**Comment by GregVido** [comment]
Yes, the light/dark theme is for Windows 11, I can look this weekend if I can make an alternative for Windows 10 if you like

**Comment by tonylu110** [comment]
> Yes, the light/dark theme is for Windows 11, I can look this weekend if I can make an alternative for Windows 10 if you like

I remember that in a previous version Windows 10 could use light/dark theme.
So this problem is an error in the light/dark theme setting

**Comment by GregVido** [comment]
Normally it is fixed for Windows 10

**Comment by tonylu110** [comment]
> Normally it is fixed for Windows 10

Okey, thanks.
Will you support Windows 10's light/dark theme, or now it supports.

**Comment by GregVido** [comment]
I don't know if it works currently, I have internet problems, I can't download a vm to test 😔
